core/rfb.js

```diff
@@ -1271,6 +1462,42 @@
                 msg_type = this._sock.rQshift8();
             }
 
+            if (this._rfb_atenikvm) {
+                switch (msg_type) {
```

Comment:
Do these conflict with existing message?

They should be in the main switch. And in rfbproto. And reported to IANA.